src/panels/config/devices/ha-config-devices-dashboard.ts

```diff
@@ -302,6 +302,14 @@ export class HaConfigDeviceDashboard extends LitElement {
                   .batteryChargingStateObj=${batteryCharging}
                 ></ha-battery-icon>
               `
+            : battery && (battery.state === "on" || battery.state === "off")
+            ? html`
+                <ha-battery-icon
+                  .hass=${this.hass!}
+                  .batteryStateObj=${battery}
+                  .batteryChargingStateObj=${batteryCharging}
+                ></ha-battery-icon>
+              `
```

Comment:
This the same as above? You can add the check if it is a binary sensor on line 296?

Reply:
+ a conditional on line 298